src/Compat.jl

```diff
@@ -739,6 +739,27 @@ if VERSION < v"1.6.0-DEV.1083"
     end
 end
 
+if VERSION < v"1.3"
+    # Note this is much less performant than real 5-arg mul!, but is provided so old versions of julia don't error at least
+
+    function _mul!(C, A, B, alpha, beta)
+        Y = similar(C)
+        mul!(Y, A, B)
+        Y .= alpha.*Y .+ beta.*C
+        return Y
```

Comment:
```suggestion
        C .= Y .* alpha .+ C .* beta
        return C
```
(1) It should write into `C`, and (2) the scalars are formally multiplied from the right (doesn't matter for `RealOrComplex`, but still).